whylog_vim/gui/windows.py

```diff
@@ -69,7 +69,13 @@ def __init__(self):
         self.windows = dict()
 
     def create_window(self, window_type, content, modifiable=False, size=None):
+        self.origin_filename = VimUIWrapper.get_current_filename()
+        self.origin_offset = VimUIWrapper.get_cursor_offset()
         self.windows[window_type] = Window(window_type, content, modifiable, size)
+        for window in six.iterkeys(self.windows):
+            if self.origin_filename.endswith(window):
```

Comment:
this will close the first window with a file that looks sort of like the target.